fendermint/actors/chainmetadata/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "fendermint_actor_chainmetadata"
+description = "Actor for storing chain metadata"
+license.workspace = true
+edition.workspace = true
+authors.workspace = true
+version = "0.1.0"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+
+[dependencies]
+cid = { workspace = true, default-features = false }
+fil_actors_runtime = { workspace = true, features = ["fil-actor"] }
+fvm_shared = { workspace = true }
+fvm_ipld_encoding = { workspace = true }
+fvm_ipld_blockstore = { workspace = true }
+num-derive = { workspace = true }
+serde = { workspace = true, features = ["derive"] }
+serde_tuple = { workspace = true }
+num-traits = { workspace = true }
+frc42_dispatch = { workspace = true }
+anyhow = { workspace = true }
+
+[features]
+fil-actor = []
```

Comment:
I don't fully understand this feature. 

At first glance I thought that because of [this conditional](https://github.com/consensus-shipyard/ipc/pull/568/files#diff-74a1adc014625e31a29cc808774f2b2351cef9055d0c1e65e5680c1782c32187R8-R11) thing it will only require the actor runtime if the target is Wasm, which would be cool. But then wouldn't `fil_actors_runtime` have to be an `optional` dependency activated by this feature, similar to how it's in the [testing](https://github.com/consensus-shipyard/ipc/blob/8f165c6db1eec7898ad5725a541f431bc41f0437/fendermint/testing/Cargo.toml#L35) crate where there are lots of optional stuff you might not need, or the [rocksdb](https://github.com/consensus-shipyard/ipc/blob/8f165c6db1eec7898ad5725a541f431bc41f0437/fendermint/rocksdb/Cargo.toml#L29-L30) crate optionally implementing blockstore and kvstore?

Reply:
I'm not sure if you don't need to declare an empty `default-features` too to make sure by default this is off.

Reply:
I made the `fil_actors_runtime` a conditional dependency when fil-actor is set